data-collection-scripts/long-calculations/longcalcs.py

```diff
@@ -0,0 +1,224 @@
+	print("\nAntigen Positives", file=f)
+	print(res.json()['features'][5]['attributes']['Count_'], file=f)
+
+	# Antibody Positives
+	print("\nAntibody Positives", file=f)
+	print(res.json()['features'][2]['attributes']['Count_'], file=f)
+	print("\n\n", file=f)
+
+
+# UT Testing
+def ut(f):
+    print("Run at: ", datetime.now(tz('US/Eastern')), "\n", file=f)
+    url = 'https://coronavirus-dashboard.utah.gov/Utah_COVID19_data.zip'
+    res = requests.get(url)
+    zipdata = BytesIO(res.content)
+    zip = zipfile.ZipFile(zipdata, 'r')
+    for zf in zip.filelist:
+      if zf.filename.startswith('Overview_Total Tests by Date'):
+        # yay, the testing file
+        title = 'Tests'
+      elif zf.filename.startswith('Overview_Number of People Tested by Date'):
+        title = 'People'
+      else:
+        title = None
+      if title:
+        title = "Metrics for {} (from {})".format(title, zf.filename)
+        print(title, "\n"+"="*len(title), file=f)
+        df = pd.read_csv(zip.open(zf.filename)).drop(columns=[' Total Daily Tests', 'Total Positive Tests', 'Daily People Tested', 'Daily Positive Tests'], errors="ignore")
+        print(df.groupby(['Test Type', 'Result']).sum(), file=f)
+        print("\n\n", file=f)
+
+
+# WA Testing
+def wa(f):
+	print("Run at: ", datetime.now(tz('US/Eastern')), "\n", file=f)
+	wa_link = 'https://www.doh.wa.gov/Portals/1/Documents/1600/coronavirus/data-tables/PUBLIC_Tests_by_Specimen_Collection.xlsx'
+	print("Link = ", wa_link, file=f)
+
+	wa = pd.read_excel(wa_link, sheet_name = 'State').filter(regex='(Positive|Negative)').drop(columns='Positive tests (%)')
+	wa.columns = [x.split()[0] for x in wa.columns]
+	print(wa.groupby(wa.columns.values, axis=1).sum().sum(), file=f)
+	print("\n\n", file=f)
+
+
+# WI PCR Testing Encounters
+def wi(f):
+	print("Run at: ", datetime.now(tz('US/Eastern')), "\n", file=f)
+	wi = pd.read_csv("https://bi.wisconsin.gov/t/DHS/views/PercentPositivebyTestPersonandaComparisonandTestCapacity/TestCapacityDashboard.csv", thousands=",")
+	print("PCR Testing Encounters: " + str(wi[wi['Measure Names'] == 'Total people tested daily']['Number of Tests'].sum()), file=f)
+	print("\n\n", file=f)
+
+
+# WV Testing
+def wv(f): 
+    print("Run at: ", datetime.now(tz('US/Eastern')).isoformat(), "\n", file=f)
+    wv = pd.read_csv("https://raw.githubusercontent.com/COVID19Tracking/covid19-datafetcher/data/wv_lab_tests.csv", thousands=",")
+    print(wv.loc[:, wv.columns != 'date'].sum(axis=0), file=f)
+    print("\n\n", file=f)
+
+
+def main():
+	state = ['de','hi','ma','me', 'mi', 'nc', 'nd', 'oh', 'tx', 'ut', 'wa', 'wi', 'wv']
+	for s in state:
+		path = '../data/' + s + '.txt'
```

Comment:
Should this be `../../data/` since we're in `/data-collection-scripts/long-calculations` and want to get to `/data`?

Reply:
Oh wow great catch! Thank you. Updating.